securing_apps/topics/oidc/java/client-authentication.adoc

```diff
@@ -66,11 +66,16 @@ The client has a secret, which needs to be known to both the adapter (applicatio
 ----
 "credentials": {
   "secret-jwt": {
-    "secret": "19666a4f-32dd-4049-b082-684c74115f28"
+    "secret": "19666a4f-32dd-4049-b082-684c74115f28",
+    "algorithm": "HS512"
   }
 }
 ----
 
+"algorithm" field specifies the algorithm for Signed JWT using Client Secret. It needs to be the one of the following values : HS256, HS384, and HS512. As for the details about them, please refer to https://tools.ietf.org/html/rfc7518#section-3.2[JSON Web Algorithms (JWA)].
```

Comment:
remove "the" before "one"

Change "As for the details about them" to "For details"

Reply:
Yes, I'll fix it as you said.